crates/shared_types/src/directive.rs

```diff
@@ -464,6 +532,14 @@ mod tests {
             })
         );
 
+        assert_eq!(
+            parse_to_directive("attr.string_list:plugins://x/y/z:artifact"),
```

Comment:
my only nit here is that using `attr.string_list` it looks like the "type system" used by starlark rules, but actually, these "strings" could also be `label_list`. So I wonder if not using the exact same term would be helpful to avoid confusion.

Maybe just `attr_list` might be better?

Reply:
Yea, the beauty of [`attr`](https://bazel.build/rules/lib/toplevel/attr) is that it's finite, and I did take that name. My thinking is that is `attr.string_list` goes well, we can keep expanding to cover `attr.int` etc. I'd be happy to add `attr.label_list` as a syntactic sugar.

Reply:
Added `attr.label_list` in 525096d2c8e7dd0fc2228985b2c42cdb754d76d9